### PR TITLE
Fix V3 scans for client-side & generate separate VSCode notices bundle

### PR DESCRIPTION
**Pull request by t1m0thyj**
* Moved the notices for `zowe-explorer-vscode` into a separate zip artifact. Previously the VSCode notices were being included with the zos components.
* Added `--ignore-scripts` flag to npm/yarn/pnpm install commands. Since the `--prod` flag was added in https://github.com/zowe/zowe-dependency-scan-pipeline/pull/145, the `prepare` script fails for some repos if it requires a dev dep (e.g. `husky`).
* Added step to install pnpm in workflow because it was missing from PATH. Not sure why the install script added to Dockerfile in https://github.com/zowe/zowe-dependency-scan-pipeline/pull/145 didn't work.
* Updated version of ORT in Dockerfile and published a snapshot with `test-timothy` tag. V15 had several errors causing it to fail parsing [Cargo](https://github.com/oss-review-toolkit/ort/issues/8480) (CLI daemon) and [PNPM](https://github.com/oss-review-toolkit/ort/issues/8987) (ZE) projects that were fixed by upgrading.

@awharn @zFernand0 @traeok Please review the [CLI](https://zowe.jfrog.io/ui/repos/tree/General/libs-snapshot-local/org/zowe/licenses/3.0.0-SNAPSHOT/zowe_licenses_cli-3.0.0-SNAPSHOT-20240927023104.zip?clearFilter=true) and [VSCode](https://zowe.jfrog.io/ui/repos/tree/General/libs-snapshot-local/org/zowe/licenses/3.0.0-SNAPSHOT/zowe_licenses_vscode-3.0.0-SNAPSHOT-20240927023104.zip?clearFilter=true) artifacts generated by the license workflow.

### CODE REVIEW

File: .github/workflows/license-generation.yml [hunk 1]
```diff
@@ -84,13 +88,13 @@ jobs:
   create-licenses:
 
     runs-on: ubuntu-latest
-  
+
     container:
-      image: zowe-docker-release.jfrog.io/ompzowe/zowecicd-license-base:latest
+      image: zowe-docker-snapshot.jfrog.io/ompzowe/zowecicd-license-base:test-timothy
```
Comment (traeok) [inline .github/workflows/license-generation.yml]: Do we still plan on using the `test-timothy` tag for the container image? No request for changes here, just wondering 😋

**Review by traeok — APPROVED** [review]
LGTM, thanks Timothy for the fixes!

Not related, but I see we have a couple "android"-related deps in the dependency tree 😂 not quite sure where that's coming from, but the licenses & dependency list seems accurate

image

File: .github/workflows/license-generation.yml (hunk 1 above)
Comment (MarkAckert) [reply .github/workflows/license-generation.yml]: Please revert the image to release/latest before merging in 👍

File: .github/workflows/license-generation.yml [hunk 2]
```diff
@@ -102,68 +106,75 @@ jobs:
...
         env:
           APP_NOTICES_SCAN: true
           APP_LICENSE_SCAN: true
           ZOWE_MANIFEST_BRANCH: ${{ env.ZOWE_RELEASE_BRANCH }}
         working-directory: ${{ env.DEPENDENCY_SCAN_HOME }}
         run: |
           # Rustup is set to default in the container, but it's not picked up in this run block
-          rustup default stable  
+          rustup default stable
+          npm install -g pnpm@8
```
Comment (MarkAckert) [inline .github/workflows/license-generation.yml]: Can we move this pnpm install to the docker build?

**Pull request closed without merging**